client/components/banner/index.jsx

```diff
@@ -65,14 +66,39 @@ class Banner extends Component {
 	};
 
 	getHref() {
-		const { href, feature, siteSlug } = this.props;
+		const { feature, plan, siteSlug } = this.props;
 
-		if ( ! href && siteSlug ) {
-			if ( feature ) {
-				return `/plans/${ siteSlug }?feature=${ feature }`;
-			}
-			return `/plans/${ siteSlug }`;
+		const baseUrl = `/plans/${ siteSlug }`;
+		let selectedPlan = 'Personal';
```

Comment:
Any good reason to default to a personal plan? I'd assume we want to default to nothing instead.

Reply:
True, for a moment I thought of it as the default current setting, but we do have it in place when it is `undefined` 👍